growth-fancy/index.php

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+			<LINK href="style.css?version=9345" rel="stylesheet" type="text/css">
```

Comment:
Should be `?version=<?php echo time(); ?>` to avoid caching, like in the original file

Reply:
Oops. Initially I copied the output page on client side. I already wondered why it is there but forgot to have a look at the other index.php pages. 😊